rpcs3/CMakeLists.txt

```diff
@@ -144,8 +144,12 @@ elseif(UNIX)
         COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_SOURCE_DIR}/bin/git $<TARGET_FILE_DIR:rpcs3>/git)
 elseif(WIN32)
     if(MSVC)
-        add_custom_command(TARGET rpcs3 POST_BUILD
-            COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_BINARY_DIR}/3rdparty/OpenAL/openal-soft/$<CONFIG>/OpenAL32.dll $<TARGET_FILE_DIR:rpcs3>)
+        cmake_policy(SET CMP0026 OLD)
```

Comment:
What is this for?

Reply:
`CMP0026`Disallow use of the LOCATION property for build targets, so I add this line.

Reply:
> Better use
> 
> ```
> COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:OpenAL::OpenAL> $<TARGET_FILE_DIR:rpcs3>
> ```
> 
> `if(MSVC)` can be removed then
> 
> and change `OpenAL\CMakeLists.txt` to
> 
> ```
> # OpenAL
> if(USE_SYSTEM_OPENAL)
> 	find_package(OpenAL CONFIG REQUIRED)
> 	add_library(3rdparty_openal INTERFACE)
> 	target_link_libraries(3rdparty_openal INTERFACE OpenAL::OpenAL)
> 	set_target_properties(OpenAL::OpenAL PROPERTIES IMPORTED_GLOBAL ON)
> else()
> ```

ok nice code

Reply:
It fails on Linux because `OpenALConfig.cmake` is not installed. I hate CMake. You need something like:
```
if(WIN32)
  find_package(OpenAL CONFIG REQUIRED)
else()
  find_package(OpenAL REQUIRED)
endif()
```

Reply:
thks. I don`t have a linux with OpenAL😂